lua/sps/items/weapons.lua

```diff
@@ -103,6 +103,14 @@ PS_WeaponProduct({
 	class = 'weapon_gauntlet'
 })
 
+PS_WeaponProduct({

```

Comment:
I'm not sure this should be a shop item. I could put it in the mystery box instead (the code for that is private, I can do it)

Reply:
Revert in the same PR. I assume the audio is fine, you can revise it later if you want to.